pkg/alertmanager/alertmanager.go

```diff
@@ -378,13 +379,14 @@ func (am *Alertmanager) ApplyConfig(userID string, conf *config.Config, rawCfg s
 	for _, ti := range conf.TimeIntervals {
 		timeIntervals[ti.Name] = ti.TimeIntervals
 	}
+	intervener := timeinterval.NewIntervener(timeIntervals)
```

Comment:
New layer wrapping the time intervals map: https://github.com/prometheus/alertmanager/pull/3556